src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -366,6 +366,7 @@ private bool Initialize(INewCommandInput commandInput)
             if (reinitFlag)
             {
                 EnvironmentSettings.Host.FileSystem.DirectoryDelete(EnvironmentSettings.Paths.HostVersionSettingsDir, true);
+                EnvironmentSettings.Host.FileSystem.CreateDirectory(EnvironmentSettings.Paths.HostVersionSettingsDir);
```

Comment:
Since we broke it once, can you please consider adding simple integration test?
Smth like:
```
> dotnet new 
(check for no error)
> dotnet new --debug:reinit
(check for no error and same output)
```
It would be good to add same test for `--debug:rebuildcache`

Reply:
Added two tests: for `--debug:reinit` and for `--debug:rebuildcache`.